<launch>
    <!-- ArUco 마커 로컬 인식 -->
    <!--
    <node
        pkg="video_sender"
        exec="aruco_localizer_node"
        name="aruco_localizer"
        output="screen" />
    -->

    <!-- ArUco 마커 위치 저장기 -->
    <!--
    <node
        pkg="video_sender"
        exec="marker_recorder"
        name="marker_recorder"
        output="screen" />
    -->

    <!-- 카메라 퍼블리셔 노드 추가 -->
    <node
        pkg="video_sender"
        exec="camera_publisher"
        name="camera_publisher"
        output="screen" />

    <!-- AI 서버로 UDP 영상 송신 -->
    <node
        pkg="video_sender"
        exec="udp_streamer_node"
        name="udp_streamer"
        output="screen">
        <param name="server_ip"   value="192.168.0.73"/>
        <param name="server_port" value="4436"/>
        <param name="roscar_id"   value="1"/>
        <param name="send_rate"   value="5.0"/>
    </node>
</launch>
